<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IkanBubbleSort</class>
 <widget class="QMainWindow" name="IkanBubbleSort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IkanBubbleSort</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>263</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ikan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonJenisAsc">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>600</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Jenis (Asc)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditJenis">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditJenis_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>357</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>451</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tambah Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Jenis</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>313</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Berat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonIkanAsc">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>600</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ikan (Asc)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBeratAsc">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>600</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Berat (Asc)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIkan">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonJenisDesc">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>640</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Jenis (Desc)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushUpdate">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>470</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBerat">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>340</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineUpdateIkan">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>398</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Tambah</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>570</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bubble Sort</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Edit Data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIkan_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>183</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ikan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hapus Data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBerat_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Program Bubbe Sort Data Ikan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ikan yang dihapus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonIkanDesc">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>640</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ikan (Desc)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>250</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Jenis</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>20</width>
      <height>361</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data Baru</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ikan yang diedit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHapus">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Hapus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBeratDesc">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>640</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Berat (Desc)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>393</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Berat</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>20</width>
      <height>361</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineHapusIkan">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
